### ISSUE TITLE
Support ipywidgets >=8

### ISSUE THREAD

**Issue by DManowitz** [Kitware/ipyvtk-simple#36]
This package currently requires ipywidgets <8.  However, versions 8.0.0 - 8.0.2 are available.

**Comment by banesullivan** [comment]
Correct, but there are issues with using ipywidgets >=8. Please reference the following:

- https://github.com/pyvista/pyvista/issues/3360
- https://github.com/pyvista/pyvista/issues/3274
- https://github.com/pyvista/pyvista/issues/3282

I'm currently working on an entirely new notebook rendering backend based on [`trame`](https://github.com/Kitware/trame) over in https://github.com/pyvista/pyvista/pull/3385 and plan on deprecating this library when that is complete. The `trame` based back-end is ***much** more performant

Unfortunately, I do not have the bandwidth to fix the issues with the latest `ipywidgets` but would welcome any community changes to fix it
